src/main/flight/mixer.c

```diff
@@ -510,26 +527,33 @@ void mixTable(pidProfile_t *pidProfile)
     if (feature(FEATURE_3D)) {
         if (!ARMING_FLAG(ARMED)) throttlePrevious = rxConfig()->midrc; // When disarmed set to mid_rc. It always results in positive direction after arming.
 
-        if ((rcCommand[THROTTLE] <= (rxConfig()->midrc - flight3DConfig()->deadband3d_throttle))) { // Out of band handling
+        throttle3Dstate_e flight3DState = get3DState(throttlePrevious);
```

Comment:
I would say the penalty is very minimal and it is only affecting 3D mode anyways. 
IMO the mixTable() got cluttered and should be separated in parts to make it more maintainable.

One of the things facing now is the throttle to float conversion, which now is done in mixTable(), while it should be done prior to that